src/main/resources/io/jenkins/plugins/gitlabbranchsource/servers/GitLabServers/config.groovy

```diff
@@ -0,0 +1,24 @@
+package io.jenkins.plugins.gitlabbranchsource.servers.GitLabServers
+
+import lib.FormTagLib
+
+def f = namespace(FormTagLib)
+
+f.section(title: descriptor.displayName) {
+
+    f.entry(title: _("GitLab Servers"),
+            help:descriptor.getHelpFile()) {
+
+        f.repeatableHeteroProperty(
+                field: "servers",
+                hasHeader: "true",
+                addCaption: _("Add GitLab Server")
+        )
+    }
+    f.advanced() {
+
+        f.entry("title": "Advanced configurations") {
+            f.textbox("default": "Will be added in later release")
```

Comment:
Same thing here about what this section is about, what is not there and what may be coming